app/util/AWS.scala

```diff
@@ -24,6 +25,15 @@ class AWSConfig @Inject() (config: Configuration) {
 
   lazy val credProvider = new AWSCredentialsProviderChain(credProviders.flatten: _*)
 
+  lazy val sessionId: String = "session" + Math.random()
+
+  lazy val kinesisArn = config.getString("aws.kinesis.streamArn").getOrElse("")
+
+  lazy val atomsCredProvider = new AWSCredentialsProviderChain(
+    new ProfileCredentialsProvider("composer"),
```

Comment:
I'm not sure what you mean. If someone wanted to use a different kinesis stream, then they'd have to change the name and arn of the stream they were using, and if they wanted to use CODE/PROD locally, they'd have to, with this setting, get composer credentials, and set a variable in the configuration file to true.